src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.tsx

```diff
@@ -151,9 +155,9 @@ function PopoverReportActionContextMenu(_props, ref) {
         isPinnedChat = false,
         isUnreadChat = false,
     ) => {
-        const nativeEvent = event.nativeEvent || {};
+        const {pageX = 0, pageY = 0} = 'nativeEvent' in event ? event.nativeEvent : {};
         contextMenuAnchorRef.current = contextMenuAnchor;
-        contextMenuTargetNode.current = nativeEvent.target;
+        contextMenuTargetNode.current = event.target as HTMLElement;
```

Comment:
Why do we change the behavior?

We switched from `event.nativeEvent.target` to `event.target` here

Reply:
This is the same element, it's just to make it more consise